apps/spruce/src/pages/adminSettings/tabs/GeneralTab/FeatureFlagsTab/getFormSchema.ts

```diff
@@ -18,6 +18,7 @@ const serviceItems: Record<string, string> = {
   podAllocatorDisabled: "Allocate pods for container tasks",
   unrecognizedPodCleanupDisabled: "Clean up unrecognized pods",
   cloudCleanupDisabled: "Cloud Provider Cleanup",
+  debugSpawnHostDisabled: "Debug spawn hosts disabled",
```

Comment:
[nit] Since the radio button says "disabled" I'd omit this just so there's no chance of interpreting it as a double negative
```suggestion
  debugSpawnHostDisabled: "Debug spawn hosts",
```

Reply:
updated! 

I didnt use the commit button because codegen was killing me 😠